package.json

```diff
@@ -26,7 +26,8 @@
 		"Other"
 	],
 	"activationEvents": [
-		"onCommand:mdmmp.showMindMap"
+		"onCommand:mdmmp.showMindMap",
+		"onCommand:mdmmp.exportSVG"
```

Comment:
I think using little camel-case is better, so can you change the command to `mdmmp.exportSvg`?